src/sssom/parsers.py

```diff
@@ -795,11 +804,17 @@ def to_mapping_set_document(msdf: MappingSetDataFrame) -> MappingSetDocument:
 def _get_mapping_set_from_df(df: pd.DataFrame, meta: Optional[MetadataType] = None) -> MappingSet:
     mapping_set = _init_mapping_set(meta)
     bad_attrs: Counter = Counter()
-    for _, row in df.iterrows():
-        mapping_dict = _get_mapping_dict(row, bad_attrs)
-        _add_valid_mapping_to_list(mapping_dict, mapping_set.mappings)
+
+    df.apply(
+        lambda row: _add_valid_mapping_to_list(
+            _get_mapping_dict(row, bad_attrs), mapping_set.mappings
+        ),
+        axis=1,
+    )
+
```

Comment:
I'm a little confused about `apply` here, does it actually alter the row? or just a different way to iterate over rows?

Reply:
So each row is passed as a param to `_get_mapping_dict()` first whose result is then passed to `_add_valid_mapping_to_list` that updates `mapping_set.mappings`. The `mapping_set` object is eventually returned by the function. I hope that makes sense?

Reply:
So basically a different way to iterate over rows. [Apparently it is more efficient](https://stackoverflow.com/a/24871316/947778) than `df.itterows()`